aggregator/src/aggregator/collect_job_driver.rs

```diff
@@ -1,5 +1,6 @@
 //! Implements portions of collect sub-protocol for DAP leader and helper.
```

Comment:
I don't agree with this file rename because several items from this module are used in the helper's `/aggregate_share` implementation, so it's not exclusive to the leader's `collect_job_driver`. However I concede that the existing name `aggregate_share.rs` is misleading in the other direction, since that resource is exclusive to the DAP helper. I don't have a better idea for naming the file right now, so I'm OK going ahead with this.

Reply:
I noticed this too. I'm going to split out the pieces that don't fit in the next PR. (the motivation for this change was basically just, the aggregation job creator is `aggregation_job_creator.rs`; the aggregation job driver is `aggregation_job_driver.rs`; the collect job driver is `aggregate_share.rs` -- it'd be better if they matched)

Reply:
Yes, I agree that harmonizing with the name schemes for everything else has value. Also, and this is rather speculative: if my plans for https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/367 come to fruition, then aggregate share handling in the helper might become asynchronous just like collection handling in the leader, at which point we might want to run something almost exactly like `collect_job_driver` in Janus helpers, so maybe a `<something>_job_driver` (where "something" is a noun that accurately captures both aggregate shares and collections) is the right direction to go.